### DISCUSSION TITLE
Sonic frr isue save config

### DISCUSSION THREAD

**Discussion by wegarc** [community/community#134307] [Programming Help]
### Body

Hi, i have the problem with frr in Sonic, When i reboot the switch lost config for frr.

Please help me!

my version is:

root@sonic:/etc/network/interfaces.d# show version

SONiC Software Version: SONiC.Edgecore-SONiC_20240425_054819_ec202111_819
Distribution: Debian 11.9
Kernel: 5.10.0-8-2-amd64
Build commit: 746f8e085
Build date: Thu Apr 25 11:28:13 UTC 2024
Built by: ubuntu@ip-10-5-1-100

Platform: x86_64-accton_as7712_32x-r0
HwSKU: Accton-AS7712-32X
ASIC: broadcom
ASIC Count: 1
Serial Number: 771232X1828033
Model Number: FP3ZZ7632014A
Hardware Revision: N/A
Uptime: 15:24:55 up 29 min,  1 user,  load average: 2.84, 2.89, 2.57

Docker images:
REPOSITORY                    TAG                                           IMAGE ID       SIZE
docker-sonic-p4rt             Edgecore-SONiC_20240425_054819_ec202111_819   81e6b2e84b1e   534MB
docker-sonic-p4rt             latest                                        81e6b2e84b1e   534MB

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by oussemamansouri** [comment]
Can you explain more?

**Comment by wegarc** [comment]
Hi mr,

I think that problem is:

frr path:

Building Configuration...
Configuration saved to /etc/frr/zebra.conf
Configuration saved to /etc/frr/bgpd.conf
Configuration saved to /etc/frr/staticd.conf
Configuration saved to /etc/frr/vrrpd.conf

Sonic path:

/etc/sonic/frr/frr.conf

Is possible to modify the frr path?

**Reply by wegarc** [reply]
this is the problem:

https://github.com/sonic-net/SONiC/issues/250
